cpp/basix/cell.cpp

```diff
@@ -16,6 +16,8 @@ xt::xtensor<double, 2> cell::geometry(cell::type celltype)
 {
   switch (celltype)
   {
+  case cell::type::point:
+    return xt::xtensor<double, 2>({{}});
```

Comment:
Should this be
```
return xt::xtensor<double, 2>({{0.0}});
```
?

Reply:
Point has 0 topological dimension so I thought should have coordinates of length 0. But I'm not certain this is sensible

Reply:
Tests pass either way, so I'm adding the 0.0 to make it length 1. Can always revise it once we have an example in DOLFINx that actually uses this functionality